src/components/Clients/ClientListTable.jsx

```diff
@@ -30,6 +30,20 @@ const ClientListTable = ({ statusType, defaultMessage }) => {
   const { state, dispatch } = useStatusNotification();
   const { userListObject } = useContext(UserListContext);
 
+  function comparePerson(a, b) {
```

Comment:
I think this is good, seems to work as intended for sorting.

Reply:
If you didn't test thoroughly there is an issue with this sorting.

<img width="678" alt="Screenshot 2023-06-21 at 12 32 00 PM" src="https://github.com/codeforpdx/PASS/assets/71395931/8893cb4b-002f-41ca-add6-35356caf9972">

The capitalization all goes first, then the lower case - so even names that are the same get split up.

---

So, one of the conversations now is how to deal with that...

1) Make the "Add Client" function force the names into the array the way we want (capitalized or not) so the array data is the way we expect and sort will work as intended

2) Make the sort function more complex to deal with this

3) Some other way

Reply:
Another conversation is: is this how we want it sorted?  Currently it's comparing the `.person` (meaning complete name first and last), I personally think that the convention when sorting by name is sorting alphabetically by last name.

This would require to change the Client List Table headers to have separate first name and last name fields instead of the combined person field.    

BUT, I feel this would be a logical step forward anyway as I think it would be nice to let the user control how the table is sorted (once it renders with this first sorting.) By way of the table header having clickable arrows that would sort somehow. I believe this to be possible with MUI table component.

Reply:
Could make a sortable table component that lets you click the header of every column to sort on that column

Reply:
> Could make a sortable table component that lets you click the header of every column to sort on that column

That’s what I was trying to describe in the paragraph above yours. 

But that’s a separate sorting. The sorting in this PR is for the initial render.